sat-manpage.sh

```diff
@@ -26,8 +26,15 @@
 
 # Text surrounded by @ is replaced by a sed command in the spec file
 sat_repository=${SAT_REPOSITORY:-@DEFAULT_SAT_REPOSITORY@}
-sat_image_tag=${SAT_TAG:-@DEFAULT_SAT_TAG@}
-sat_image=${SAT_IMAGE:-$sat_repository:$sat_image_tag}
+sat_version_file="/opt/cray/etc/sat/version"
+if [[ -f "$sat_version_file" ]]; then
+    sat_image_name=${SAT_IMAGE_NAME:-@DEFAULT_SAT_IMAGE_NAME@}
+    sat_image_tag="${SAT_TAG:-$(cat "$sat_version_file")}"
+else
+    sat_image_name=${SAT_IMAGE_NAME:-@CSM_SAT_IMAGE_NAME@}
+    sat_image_tag="${SAT_TAG:-csm-latest}"
+fi
+sat_image=${SAT_IMAGE:-$sat_repository/${sat_image_name}:$sat_image_tag}
```

Comment:
It seems like there is some inconsistent use of `${variable}` vs. just `$variable` for variable expansion. For the sake of readability, it would be nice to be consistent.

Reply:
Should be fixed in 82e4438e7ef1dd0bc7e6c7efb530a7a7167f65ef. I noticed there was a little inconsistency with variable quoting too. I don't think it's actually necessary inside of parameter expansions but I think I have it more consistent now.